invenio_app_rdm/config.py

```diff
@@ -767,3 +783,17 @@ def files_rest_permission_factory(obj, action):
     ("invenio_app_rdm/default_static_page.html", "Default"),
 ]
 """List of available templates for pages."""
+
+FILES_REST_CHECKSUM_VERIFICATION_URI_PREFIXES = [
+    # 'root://eospublic'
+]
+"""URI prefixes of files their checksums should be verified"""
+
+FILES_INTEGRITY_REPORT_TEMPLATE = "invenio_app_rdm/emails/files_integrity_report.html"
+"""Files integrity report template"""
+
+FILES_INTEGRITY_REPORT_SUBJECT = "Files integrity report"
+"""Files integrity report subject"""
+
+ADMIN_EMAIL_RECIPIENT = "info@inveniosoftware.org"
```

Comment:
Please mention this (configuration of files check and admin email) to the notes for the upgrade/release: https://github.com/inveniosoftware/docs-invenio-rdm/issues/447
Can you also add in the notes anything else that was already done but forgotten?